libcontainer/cgroups/fs/cpuset.go

```diff
@@ -14,6 +15,44 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+var (
+	cpusetLock     sync.Mutex
+	cpusetNoPrefix bool
+	cpusetFastPath bool
+)
+
+func getCpusetFile(path string, name string) string {
+	cpusetLock.Lock()
+	defer cpusetLock.Unlock()
+
+	// only allowed cgroup to mount with no prefix is v1 cpuset
+	// https://github.com/torvalds/linux/blob/2e1b3cc9d7f790145a80cb705b168f05dab65df2/kernel/cgroup/cgroup-v1.c#L1070
+	// it cannot be mounted with and without prefix at the same time
+	// it happens to be used on Android.
+
+	if !cpusetFastPath {
+		fd, err := cgroups.OpenFile(path, "cpuset."+name, unix.O_RDONLY)
```

Comment:
This is kind of expensive. You can just use `unix.Access(path, unix.F_OK)` instead.

Reply:
done